java/FoodTruckApp/src/main/java/com/my/foodTruckApp/Appetizer/AppetizerRepository.java

```diff
@@ -66,4 +66,15 @@ public void deleteAppetizerById(Integer id) throws ResponseStatusException {
         jdbcTemplate.update(deleteSql, id);
         log.info("deleted appetizer with id: " + id);
     }
+
+    public void createAppetizerOrder(Integer orderId, Integer appetizerId) throws ResponseStatusException {
+        String appetizerSql = "INSERT INTO appetizer_ordered (order_id, appetizer_id) VALUES (?, ?) RETURNING * ";
+        AppetizerOrdered newAppetizerOrdered = jdbcTemplate.queryForObject(
+                appetizerSql,
+                new BeanPropertyRowMapper<>(AppetizerOrdered.class),
+                orderId,
+                appetizerId
+        );
```

Comment:
you are mapping this into a `AppetizerOrdered` but not doing anything with it.  Let's return it on the method instead so it can be used if desired